### PR TITLE
added acr quickstart

### PR DESCRIPTION
**Pull request by tw3lveparsecs**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [x] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog
* added quickstarts\microsoft.containerregistry\container-registry-with-policies-and-diagnostics

### CODE REVIEW

File: quickstarts/microsoft.containerregistry/container-registry-with-policies-and-diagnostics/main.bicep [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+  'Premium'
+  'Classic'
+])
+param acrSku string = 'Basic'
+
+@description('Enable System Assign Identity for Azure Container Registry')
+param enableSystemIdentity bool = false
+
+@description('The list of user identity resource ids to associate with the Azure Container Registry')
+param userAssignedIdentities object = {}
```
Comment (bmoore-msft) [inline quickstarts/microsoft.containerregistry/container-registry-with-policies-and-diagnostics/main.bicep:22]: can we add metadata to this param to show the schema of the object that's expected for the param value?

File: quickstarts/microsoft.containerregistry/container-registry-with-policies-and-diagnostics/main.bicep [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+  properties: {
+    adminUserEnabled: enableAdminUser
+    publicNetworkAccess: publicNetworkAccess
+    zoneRedundancy: zoneRedundancy
+    networkRuleBypassOptions: networkRuleBypassOptions
+    networkRuleSet: !empty(ipRules) ? {
+      defaultAction: ipRules.defaultAction
+      ipRules: ipRules.ipRules
+    } : {}
+    policies: {
```
Comment (bmoore-msft) [inline quickstarts/microsoft.containerregistry/container-registry-with-policies-and-diagnostics/main.bicep:114]: can't this just be?
```
policies: policies
```

**Review by bmoore-msft — CHANGES_REQUESTED** [review]

File: quickstarts/microsoft.containerregistry/container-registry-with-policies-and-diagnostics/main.bicep (hunk 2 above)
Comment (tw3lveparsecs) [reply quickstarts/microsoft.containerregistry/container-registry-with-policies-and-diagnostics/main.bicep]: yes has been updated

File: quickstarts/microsoft.containerregistry/container-registry-with-policies-and-diagnostics/main.bicep (hunk 1 above)
Comment (tw3lveparsecs) [reply quickstarts/microsoft.containerregistry/container-registry-with-policies-and-diagnostics/main.bicep:25]: done

**Pull request merged**